usecases/inDevelopment/cross_consumption.json

```diff
@@ -0,0 +1,1620 @@
+      "createServiceKeys": [
+        "service-manager-container"
+      ]
+    },
+    {
+      "name": "slis",
+      "plan": "standard",
+      "category": "SERVICE",
+      "targetenvironment": "kymaruntime",
+      "createServiceKeys": [
+        "slis-standard"
+      ]
+    },
+    {
+      "name": "data-custodian",
+      "plan": "default",
+      "category": "SERVICE",
+      "targetenvironment": "kymaruntime",
+      "createServiceKeys": [
+        "data-custodian-default"
+      ]
+    },
+    {
+      "name": "spatialservices",
+      "plan": "lite",
+      "category": "SERVICE",
+      "targetenvironment": "kymaruntime",
+      "createServiceKeys": [
+        "spatialservices-lite"
+      ]
+    },
+    {
+      "name": "credstore",
+      "plan": "standard",
+      "category": "SERVICE",
+      "targetenvironment": "kymaruntime",
+      "createServiceKeys": [
+        "credstore-standard"
+      ]
+    },
+    {
+      "name": "business-rules",
+      "plan": "standard",
+      "category": "SERVICE",
+      "targetenvironment": "kymaruntime",
+      "createServiceKeys": [
+        "business-rules-standard"
+      ]
+    },
+    {
+      "name": "conversational-ai",
+      "plan": "standard",
+      "category": "SERVICE",
+      "targetenvironment": "kymaruntime",
+      "parameter": {
+        "admin_email": "akanksha.sherrie.kullu@sap.com",
+        "organization_slug": "sap1234"
+      },
+      "createServiceKeys": [
+        "conversational-ai-standard"
+      ]
+    },
+    {
+      "name": "transport",
+      "plan": "standard",
+      "category": "SERVICE",
+      "targetenvironment": "kymaruntime",
+      "createServiceKeys": [
+        "transport-standard"
+      ]
+    },
+    {
+      "name": "cloud-logging",
+      "plan": "standard",
+      "category": "SERVICE",
+      "targetenvironment": "kymaruntime",
+      "parameter": {
+        "retentionPeriod": "7"
+      },
+      "createServiceKeys": [
+        "cloud-logging-standard"
+      ]
+    },
+    {
+      "name": "alert-notification",
+      "plan": "standard",
+      "category": "SERVICE",
+      "targetenvironment": "kymaruntime",
+      "createServiceKeys": [
+        "alert-notification-standard"
+      ]
+    },
+    {
+      "name": "identity",
+      "plan": "application",
+      "category": "SERVICE",
+      "targetenvironment": "kymaruntime",
+      "createServiceKeys": [
+        "identity-application"
+      ]
+    }
+  ],
+  "enablecrossconsumptiontest": false,
+  "api_resource_uri": [
+    {
+      "servicename": "ui5-flexibility-personalization",
+      "plan": "personalization",
+      "testapi": "/sap/bc/lrep/actions/getcsrftoken",
+      "apiBaseUrlPathKey": "data.endpoints.personalization",
+      "oauthurlPathkey": "data.uaa.url",
+      "clientId_basicUN_PathKey": "data.uaa.clientid",
+      "clientSecret_basicPW_PathKey": "data.uaa.clientsecret",
+      "authMethod": "oauth",
+      "APITest": "yes",
+      "serviceID": "SERVICE-204"
```

Comment:
Wouldn't it make sense to add this as an object to the service definition in analogy to `createdServiceKeys`